contracts/NodeDelegator.sol

```diff
@@ -35,12 +35,30 @@ contract NodeDelegator is INodeDelegator, LRTConfigRoleChecker, PausableUpgradea
 
     /// @dev The EigenPod is created and owned by this contract
     IEigenPod public eigenPod;
-    /// @dev Tracks the balance staked to validators and has yet to have the credentials verified with EigenLayer.
-    /// call verifyWithdrawalCredentials to verify the validator credentials on EigenLayer
+    /* @dev Tracks the balance staked to validators and has yet to have the credentials verified with EigenLayer.
+     * Calling verifyWithdrawalCredentials acts as a message from BeaconChain to the execution chain informing the latter
+     * of the validator credentials and balance on the Beacon Chain. Once this is done the EigenPod (and other EigenLayer contracts)
+     * mint shares on the (EigenLayer's) Beacon Chain Strategy (restaking). 
+     */
     uint256 public stakedButNotVerifiedEth;
 
+    /* @dev the number of SSV validators that are in the process of exiting. That means that the exit
+     * message has already been posted by the validator and the validator is in one of the 3 possible
+     * stages:
+     *  1. The exit queue. Each epoch up to 16 validators can exit. See this page for information 
+     *     regarding the current state of the queue: https://www.validatorqueue.com/ . At the time
+     *     of writing the first stage passes in a couple of hours.
+     *  2. Validator has exited and after ~27 hours funds are marked as withdrawable.
+     *  3. Funds sweeping beacon chain process claims the funds and sends them to the EigenPod
+     * 
+     *  Call `removeSsvValidator` once the validator has completely exited (all 3 stages)
+     */
+    uint256 public validatorsExiting;
```

Comment:
What reads the `validatorsExiting` storage variable? I see its incremented in `exitSsvValidator` and decremented in `removeSsvValidator` but I can't see any function that uses `validatorsExiting`.

Reply:
will add this logic in and ping you. It will be considered when accounting for (W)ETH balances.